tools/offcputime.py

```diff
@@ -96,12 +96,11 @@ def signal_ignore(signal, frame):
 BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
 
 int oncpu(struct pt_regs *ctx, struct task_struct *prev) {
-    u32 pid;
+    u32 pid = prev->pid;
```

Comment:
I see. Will we be wasting time doing lookup for current_pid down in lines 109-111 that is not going to succeed? I guess that can be an optimization for the future?
Anyway I'm fine with it as-is.
@brendangregg